topic:
  # Input
  twist: $(arg ROBOT)/cmd_delta
  joint_states: $(arg ROBOT)/joint_states
  # Output
  trajectory: $(arg ROBOT)/controller/command
move_group_name: arm
base_frame_id: $(arg ROBOT)/base_link
ee_link_name: hand_camera
# Discard velocity field in JointState
discard_velocity: true
# Discard effort field in JointState
discard_effort: true
timeout:
  fresh_twist: 0.25
  # When to update to actual joint values
  internal_joint_state: 3
publish_rate: 50
trajectory_duration: 0.1
# Set to 0 to disable checking
joint_limits:
  speed: 1.0
  angle_change: 0.5
